lex.go

```diff
@@ -287,7 +287,14 @@ func (l *lexer) run() {
 // state functions
 func lexText(l *lexer) stateFn {
 	for {
-		if i := strings.IndexByte(l.input[l.pos:], l.leftDelim[0]); i == -1 {
+		// without breaking the API, this seems like a reasonable workaround to correctly parse comments
```

Comment:
you still haven't rebased your branch on top of the current master of CloudyKit/jet

Reply:
resolved, forced push, which might complicate matters.